recursive_art.py

```diff
@@ -137,10 +250,11 @@ def generate_art(filename, x_size=350, y_size=350):
         x_size, y_size: optional args to set image dimensions (default: 350)
     """
     # Functions for red, green, and blue channels - where the magic happens!
-    red_function = ["x"]
-    green_function = ["y"]
-    blue_function = ["x"]
-
+    red_function = build_random_function(9, 15)
+    green_function = build_random_function(9, 15)
+    blue_function = build_random_function(9, 15)
+    x = random.uniform(0, 1)
```

Comment:
what is this line of code doing? Perhaps add an inline comment explaining why you added this line of code.